### PR TITLE
Add Showtime

### LINKED ISSUE #3494: Request: Showtime Icon
**Name:** Showtime
**Website:** showtime.com
**Alexa rank:** 20,878
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by adamalston**
showtime

Closes #3494 

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Source is the company that created the Showtime logo

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6600,6 +6600,11 @@
             "hex": "2D2E83",
             "source": "https://www.showpad.com/"
         },
+        {
+            "title": "Showtime",
+            "hex": "B10000",
+            "source": "https://www.cghnyc.com/work/project/showtime"
```
Comment (adamrusted) [inline _data/simple-icons.json:6606]: Could you look to update the source to https://www.sho.com/ - just so it's a direct link to the brand's site for future reference?

File: icons/showtime.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Showtime icon</title><path d="M11.997 0C8.33 0 5.047 1.648 2.846 4.241a2.982 2.982 0 01.612.091c2.8.712 2.8 3.624 2.8 4.538H3.65s0-2.087-1.304-2.087c-.293 0-.51.11-.677.245-.073.06-.236.254-.312.548-.15.588-.142 1.593.99 2.598 1.485 1.32 4.434 2.543 4.434 5.674 0 2.429-1.722 3.724-3.21 3.942 0 0-.228.043-.646.064C5.124 22.393 8.374 24 11.997 24c3.647 0 6.91-1.63 9.11-4.197-2.46-.245-4.414-3.64-4.414-7.803s1.953-7.558 4.414-7.803A11.965 11.965 0 0011.997 0zM7.823 4.696h2.609v6h2.609v-6h2.608v15.13h-2.608v-6.522h-2.609v6.522H7.823zM21.65 6.783A2.348 5.217 0 0019.302 12a2.348 5.217 0 002.347 5.217A2.348 5.217 0 0023.997 12a2.348 5.217 0 00-2.348-5.217zM.004 11.217s-.035 2.466.366 3.653c.597 1.76 1.267 2.347 2.385 2.347.82 0 1.416-.521 1.416-1.63 0-1.24-1.537-1.976-2.6-2.78-1.003-.758-1.567-1.59-1.567-1.59z"/></svg>
```
Comment (adamrusted) [inline icons/showtime.svg:1]: Though there are some inconsistencies with the icon available [at source](https://www.sho.com/www/sho/lib/assets/svg/showtime-logo.svg), there appear to be some fairly significant differences between your version (red) and source (black) - would you mind taking another look at this one?

image

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for this @adamalston - there's a couple quick changes required on this on if you could take a look?

File: icons/showtime.svg (hunk 2 above)
Comment (adamalston) [reply icons/showtime.svg:1]: I see the inconsistency that you're talking about (artifact in the inner-part of the 'S'). I tried to simplify the path and it distorted some parts of the svg. Therefore, I pulled from an svg elsewhere online.

I've just tried to pull an svg from [Wikimedia](https://commons.wikimedia.org/wiki/File:Showtime.svg) as it doesn't appear to have the artifacts. However, this is the output I am getting from SVGOMG: 
```
<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24"><g clip-path="url(#a)" transform="matrix(.13395 0 0 .13395 34.934 18.657)"><path d="M-133.974-48.45c0-35.896 13.283-58.527 29.142-60.921-15.858-19.489-42.345-29.913-66.396-29.913-26.486 0-50.274 11.499-66.661 29.751 15.636 3.103 20.949 16.226 20.949 36.524h-19.124c0-11.986-2.494-18.191-9.592-18.191-3.186 0-5.659 1.623-7.119 4.381-.203.446-.427.913-.628 1.379-.549 1.543-.853 3.326-.853 5.293 0 24.823 40.763 23.586 40.763 61.752 0 14.887-11.417 28.777-24.274 28.777 16.893 18.455 39.972 29.508 66.539 29.508 25.836 0 49.098-10.932 65.442-28.433-15.067-3.793-28.188-25.474-28.188-59.907m-8.519 59.137h-20.503v-50.983h-22.146v50.983h-20.543v-117.664h20.503v47.394h22.186v-47.394h20.503z"/><path d="M-243.099-8.041c5.719 0 9.774-5.029 9.774-12.351 0-15.189-13.264-18.109-27.479-30.946-.203 25.98 7.848 43.297 17.705 43.297M-113.599-50.349c0-19.813 5.76-39.627 16.285-39.627 12.513 0 15.677 30.623 15.677 40.295 0 21.233-6.692 40.377-15.677 40.377-10.79.002-16.285-20.175-16.285-41.045M91.16-105.345H4.727l-8.193 67.209h-.324l-9.167-67.209h-18.801l-9.958 67.209h-.323l-7.402-67.209h-18.962L-53.132 9.116h18.313l10.302-72.5h.305l9.979 72.5H4.728L17.382-87.66h17.725V9.116h20.909V-87.66H75.14V9.116h20.909v-114.461zM106.306-105.345h27.823l9.309 58.67h.324l9.815-58.67h27.154V9.116h-18.638v-83.918h-.324L147.615 9.116h-11.56l-13.993-83.918h-.305V9.116h-15.452zM191.006-105.345h47.272v17.685h-26.364v29.244h19.429v18.658h-19.429V-8.57h27.479V9.116h-48.388v-114.461z"/></g></svg>
```

Any assistance here would be appreciated

Comment (adamrusted) [reply icons/showtime.svg:1]: @adamalston I've rebuilt from the Wikimedia link and got the below path. Could you update the source to _that_ page if that's the source we're now using? You're more than welcome to submit the below path, but we'll need another @simple-icons/maintainers to sign off on it.

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Showtime icon</title><path d="M16.99 12.167c0-4.808 1.779-7.84 3.903-8.16C18.769 1.397 15.221 0 11.999 0 8.451 0 5.265 1.54 3.07 3.985c2.094.416 2.806 2.174 2.806 4.892H3.314c0-1.605-.334-2.436-1.284-2.436-.427 0-.758.217-.954.587-.027.06-.057.122-.084.184a2.115 2.115 0 0 0-.114.71c0 3.324 5.46 3.159 5.46 8.27 0 1.995-1.53 3.855-3.252 3.855C5.35 22.52 8.441 24 12 24c3.46 0 6.577-1.464 8.766-3.808-2.018-.509-3.776-3.413-3.776-8.025zm-1.142 7.921h-2.746V13.26h-2.967v6.83H7.384V4.327h2.746v6.348h2.972V4.327h2.746v15.761zM2.372 17.58c-1.32 0-2.399-2.32-2.372-5.8 1.905 1.72 3.681 2.11 3.681 4.145 0 .981-.543 1.655-1.309 1.655zM24 12.002c0 2.844-.896 5.409-2.1 5.409-1.445 0-2.181-2.703-2.181-5.498 0-2.654.771-5.308 2.181-5.308 1.676 0 2.1 4.102 2.1 5.397z"/></svg>
```

Comment (adamalston) [reply icons/showtime.svg]: Done, thanks!

**Comment by adamrusted** [comment]
Thanks for updating @adamalston - just need another @simple-icons/maintainers to sign this one off now 👌🏻

**Review by service-paradis — APPROVED** [review]
LGTM 📺 🎉 
Thanks adam^2

_It is the first time I'm not scared to mention @adamalston instead of @adamrusted 😅_

**Pull request merged**